fdb-java-annotations/src/main/java/com/apple/foundationdb/annotation/HeuristicPlanner.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * HeuristicPlanner.java
```

Comment:
This annotation has no semantic implications. However, I started annotating methods, constructors, etc. in the code that solely are being called by the heuristic planner.